### PR TITLE
Move OPL macros to PG

### PR DESCRIPTION
**Pull request by pstaabp**
This is a companion to openwebwork/PG#724 and just updates the `defaults.config` file to point to the new locations of macros. 

You will need to checkout openwebwork/PG#724 and openwebwork/webwork-open-problem-library#949

An alternative to checking out the OPL branch you can set in `conf/localOverrides.conf` the following:
```
$pg{directories}{macrosPath} = [
   ".",                     # search the problem file's directory
   $courseDirs{macros},
   $pg{directories}{macros},
   "$pg{directories}{macros}/answers",
	 "$pg{directories}{macros}/capa",
	 "$pg{directories}{macros}/contexts",
	 "$pg{directories}{macros}/core",
	 "$pg{directories}{macros}/graph",
	 "$pg{directories}{macros}/math",
	 "$pg{directories}{macros}/misc",
	 "$pg{directories}{macros}/parsers",
	 "$pg{directories}{macros}/ui",
	 "$courseDirs{templates}/Library/macros/Alfred",
         "$courseDirs{templates}/Library/macros/BrockPhysics",
         "$courseDirs{templates}/Library/macros/CollegeOfIdaho",
         "$courseDirs{templates}/Library/macros/Dartmouth",
         "$courseDirs{templates}/Library/macros/Mizzou",
         "$courseDirs{templates}/Library/macros/WHFreeman",
         "$courseDirs{templates}/Library/macros/Wiley",
];
```

Note: there are other macros in the OPL macros directory that are not being used in the OPL.  (For example, UW-Stout). These directories should be added to the above list if desired.

### CODE REVIEW

File: conf/defaults.config [hunk 1]
```diff
@@ -1115,26 +1115,16 @@ $pg{directories}{macrosPath} = [
...
+   "$pg{directories}{macros}/answers",
+	 "$pg{directories}{macros}/capa",
+	 "$pg{directories}{macros}/contexts",
+	 "$pg{directories}{macros}/core",
+	 "$pg{directories}{macros}/graph",
+	 "$pg{directories}{macros}/math",
+	 "$pg{directories}{macros}/misc",
+	 "$pg{directories}{macros}/parsers",
+	 "$pg{directories}{macros}/ui",
+	 "$courseDirs{templates}/Library/macros/univ",
```
Comment (drgrice1) [inline conf/defaults.config:1127]: There seems to be an inconsistency in the indentation here.  All of these lines should be at the same indent level.

**Review by drgrice1 — APPROVED** [review]
This looks good other than a minor indentation issue.

@pstaabp:  Note that I rebased this pull request onto develop.  So make sure to reset your local branch.  It had a duplicate of one of the commits in #1792 in it that wasn't quite the same as that one.

File: conf/defaults.config (hunk 1 above)
Comment (pstaabp) [reply conf/defaults.config]: It was a mix of tabs and spaces.  We need to do a perltidy of webwork2.

Comment (drgrice1) [reply conf/defaults.config]: Agreed!

**Review by drgrice1 — APPROVED** [review]
Once https://github.com/openwebwork/pg/pull/724 goes in, this is ready to go also.

**Review by drdrew42 — APPROVED** [review]

**Pull request merged**